api/src/util/time.ts

```diff
@@ -0,0 +1,29 @@
+import ms from 'ms';
+
+export function getSecondsSinceEpoch(): number {
+  const now = Date.now();
+  return Math.floor(now / 1000);
+}
+
+export function addTime(base: string, amount: string, unit: string): number {
+  const baseMillis = getTimeAsMilliSeconds(base);
+  const amountMillis = getTimeAsMilliSeconds(amount);
+  const calculatedMillis = baseMillis + amountMillis;
```

Comment:
1. 두 개의 시간을 단순히 더하는 함수이면, 두 피연산자에게는 순서가 상관 없는 것 아닌가요? (base, amount 라는 역할이 필요한 지 궁금합니다.) 
2. `unit` 의 타입이 string 인 것 보다 's', 'm', 'h' 등 사용되는 리터럴의 enum 으로 정의되면 어떨까요?

3. 피연산자로 전달받는 time 인자가 string 리터럴인데 잘못된 형식의 리터럴 (ex. `10k` )이라면 어떻게 처리되는 건가요?
이 함수를 사용하는 쪽에서 잘못된 값을 전달하는 경우를 막아주기 위해  숫자amount 와 unit 을 나누어 전달받고 함수 내에서 리터럴로 만들어 사용하는 게 나을것 같습니다.

```suggestion
type Unit =  'ms' | 's' | 'm' | 'h'
type Time = {amount: number, unit: Unit}
export function addTimes(times: [Time, Time], targetUnit: Unit): number {
  const calculatedMillis = times.reduce((acc, time) => acc + getTimeAsMilliSeconds(`${time.amount}+`${time.unit}`), 0);
```

Reply:
[구두 논의 내용]
@2wheeh 께서 제안해주신대로 구현 내용을 typesafe 하게 처리해줘야 할 필요성을 느꼈습니다. 다만, typesafe 한 처리를 위해 구현 내용을 바꿀 경우, 사용성이 저해될 우려가 있었습니다. 결국, 근본적으로 간단한 시간 연산을 위해 사용성과 성능 그리고 구현의 복잡함을 감수해야 되는가에 대한 의문이 들었고, @isutare412 께서 이 정도의 구현 내용은 그때 그때 처리해도 좋지 않겠냐는 의견이 나와 구현된 time 에 관한 테스트 및 구현은 삭제 조치하게 되었습니다.

Reply:
관련 패키지 삭제: https://github.com/MovieReviewComment/Mr.C/pull/82/commits/e4a44c396b26d6d4087115fa10d17caffc38c2d6
구현 내용 삭제: https://github.com/MovieReviewComment/Mr.C/pull/82/commits/c1a4cd41a6140b9e3ec4ad6ef551e44e66973e50

두 가지로 분류하여 반영되었습니다.

Reply:
빠른 반영 감사합니다 ~